styles/jisho/catppuccin.user.css

```diff
@@ -0,0 +1,449 @@
+      );
+    }
+
+    nav .links {
+      .color_theme_picker--wrapper:hover {
+        background-color: @mantle;
+      }
+
+      .color_theme_picker--choices {
+        background-color: @mantle;
+
+        li a:hover {
+          background-color: @crust;
+        }
+      }
+    }
+
+    form.search {
+      .input_methods,
+      .input_method_button h4 {
+        color: @subtext0;
+      }
+
+      .input_method_button:hover {
+        background-color: @surface0;
+      }
+
+      .main {
+        background-color: @surface0;
+        box-shadow: none;
+      }
+
+      .inner {
+        background-color: @surface1;
+
+        input {
+          color: @text;
+        }
+      }
+
+      .search_type {
+        border-right-color: @overlay0;
+        color: @subtext0;
+      }
+
+      .submit {
+        background-color: @overlay1;
+        color: @text;
+      }
+
+      .search-form_clear-button {
+        background-color: transparent; // ! Unsure if this needs a background, also extends beyond the input
+        color: @subtext0;
+      }
+
+      .results,
+      #radical_area .results {
+        .result_label {
+          color: @text;
+        }
+
+        .result {
+          background-color: @surface1;
+
+          &:hover {
+            color: @text;
+          }
+        }
+
+        each(range(8), {
+          .g@{value} {
+            color: @blue;
+          }
+        });
+      }
+
+      #radical_area {
+        .radical_table {
+          background-image: repeating-linear-gradient(
+            180deg,
+            @surface1,
+            @surface1 32px,
+            @surface2 32px,
+            @surface2 64px
+          );
+
+          .number {
+            background-color: @text;
+            color: @base;
+          }
+
+          .radical {
+            background-image: none; // ! some radicals are rendered with a background image by default
+            color: @text;
+
+            &.available:hover {
+              background-color: @overlay0;
+            }
+
+            &.selected {
+              border-color: @text;
+              background-color: mix(@base, @yellow, 50%);
+            }
+          }
+        }
+
+        &.combined_mode {
+          // ! what is combined mode?
+          .reset_radicals {
+            color: @subtext0;
+
+            &:hover {
+              color: @text;
+              background-color: @overlay0;
+            }
+          }
+        }
+      }
+
+      .handwriting {
+        .panel {
+          background-color: @surface1;
+        }
+
+        .pencil-icon {
+          color: @surface2 !important;
+        }
+      }
+    }
+
+    .speech_area_active #speech_button,
+    .radical_area_active #radical_button,
+    .handwriting_area_active #handwriting_button {
+      border-bottom-color: @accent-color;
+    }
+
+    #what_is_this {
+      color: unset;
+    }
+
+    .japanese_word__furigana-invisible {
+      opacity: 0;
+    }
```

Comment:
is there a need for this?

Reply:
I guess this is a mishap on Jisho's side. On the light theme the furigana is hidden by setting `color` to white, but this is not correctly handled on the dark theme.
![grafik](https://github.com/user-attachments/assets/eb8c367d-3ac8-48db-9856-cfcd99616e40)
![grafik](https://github.com/user-attachments/assets/112e0ce3-971b-4eaa-bdef-194ae43b4d77)

Reply:
Added a comment in [`ba3feef` (#1264)](https://github.com/catppuccin/userstyles/pull/1264/commits/ba3feef8b96e23b4353eb1b6dfa18579fe186ffb)